ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+        <options>no_full_log</options>
+        <group>SecurityComplianceInsights</group>
+    </rule>
+
+    <rule id="91571" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^43$</field>
+        <description>Office 365: Events related to the detection in the Transport pipeline of email messages that have been tagged (manually or automatically) with sensitivity labels.</description>
+        <options>no_full_log</options>
+        <group>MIPLabel</group>
+    </rule>
+
+    <rule id="91572" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^44$</field>
+        <description>Office 365: Workplace Analytics events.</description>
+        <options>no_full_log</options>
+        <group>WorkplaceAnalytics</group>
+    </rule>
+
+    <rule id="91573" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^45$</field>
+        <description>Office 365: Power Apps events.</description>
+        <options>no_full_log</options>
+        <group>PowerAppsApp</group>
+    </rule>
+
+    <rule id="91574" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^46$</field>
+        <description>Office 365: Subscription plan events for Power Apps.</description>
+        <options>no_full_log</options>
+        <group>PowerAppsPlan</group>
+    </rule>
+
+    <rule id="91575" level="12">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^47$</field>
+        <description>Office 365: Phishing and malware events for files in SharePoint, OneDrive for Business, and Microsoft Teams from Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>ThreatIntelligenceAtpContent</group>
+    </rule>
+
+    <rule id="91576" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^48$</field>
+        <description>Office 365: Events related to data classification content explorer.</description>
+        <options>no_full_log</options>
+        <group>LabelContentExplorer</group>
+    </rule>
+
+    <rule id="91577" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^49$</field>
+        <description>Office 365: Events related to the Patients application in Microsoft Teams for Healthcare.</description>
+        <options>no_full_log</options>
+        <group>TeamsHealthcare</group>
+    </rule>
+
+    <rule id="91578" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^50$</field>
+        <description>Office 365: Events related to the MailItemsAccessed mailbox auditing action.</description>
+        <options>no_full_log</options>
+        <group>ExchangeItemAggregated</group>
+    </rule>
+
+    <rule id="91579" level="3">
```

Comment:
I think this level should be higher since these alerts are related to spam protection.

Reply:
Done